CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 ### Changed
 - Volume project download zips are reorganized to contain a zipfile for each annotation (that in turn contains a data.zip and an nml file). [#4167](https://github.com/scalableminds/webknossos/pull/4167)
 - Team managers and admins can now get tasks that they had previously cancelled. [#4088](https://github.com/scalableminds/webknossos/pull/4088)
+- Automatically turning off recording for arbitrary modes when switching to orthogonal mode to prevent accidental creation of nodes when switching back to any arbitrary mode. [#4211](https://github.com/scalableminds/webknossos/pull/4211)
```

Comment:
`Arbitrary mode` is probably hard to understand for our users, let's use the terms they know from the UI :)

```suggestion
- Recording is now automatically turned off when switching from flight/oblique to orthogonal mode to prevent accidental node creation when switching back later. [#4211](https://github.com/scalableminds/webknossos/pull/4211)
```